plugins/redis/redis.cpp

```diff
@@ -591,10 +646,39 @@ template<class type> void SyncRGet(ICodeContext * ctx, const char * options, con
     Owned<Connection> master = Connection::createConnection(ctx, cachedConnection, options, database, password, _timeout, cacheConnections);
     master->get(ctx, key, returnSize, returnValue);
 }
+void Connection::strto(const char * str, double & ret, const char * key)
```

Comment:
hyper-picky:  strto is a strange name since it begs the question to what.  fromstr would read much more naturally.